corehq/apps/reports/v2/endpoints/case_properties.py

```diff
@@ -25,6 +29,18 @@ def existing_slugs(self):
     def excluded_slugs(self):
         return ['name'] + self.existing_slugs
 
+    @property
+    def search(self):
+        return self.data.get('search', '')
+
+    @property
+    def page(self):
+        return int(self.data.get('page', 1))
+
+    @property
+    def limit(self):
+        return int(self.data.get('limit', DEFAULT_RESULTS_LIMIT))
+
     @property
     @memoized
     def case_properties(self):
```

Comment:
This seems to be copy/pasted from the other CLE, do you think it would be worthwhile to pull it out into a function that both use?